packages/utils/telemetry-utils/src/test/config.spec.ts

```diff
@@ -178,6 +178,23 @@ describe("Config", () => {
         assert.equal(config1.getBoolean("featureEnabled"), false); // from settings1.BreakGlass
     });
 
+    it("Versioned entries", () => {
+        const settings1 = {
+            entry: "1",
+            entry_1: "2",
+            entry2: "2",
+            entry2_1: "3",
+        };
+        settings1[`entry_${pkgVersion}`] = "3";
```

Comment:
this test is actually a good example of the problem that would be faced with this. if we hard code pkgVersion it will behave more like a real setting which is statically defined in a configuration system. However, this test will break at the next version bump, just like a real configuration would get a different value after a version bump

Reply:
the goal of a change should a change like this should be to write a test, where the configuration is hard coded, but the correct bhe